plugins/telink-dfu/meson.build

```diff
@@ -8,6 +8,7 @@ plugin_builtins += static_library('fu_plugin_telink_dfu',
     'fu-telink-dfu-archive.c',
     'fu-telink-dfu-common.c',
     'fu-telink-dfu-ble-device.c',
+    'fu-telink-dfu-hid-device.c',
     'fu-telink-dfu-plugin.c',
   ],
   include_directories: plugin_incdirs,
```

Comment:
can you add the emulation data and the link to the firmware file in a device test please? e.g. see https://github.com/fwupd/fwupd/blob/main/plugins/cros-ec/meson.build#L21 and https://github.com/fwupd/fwupd/blob/main/plugins/cros-ec/tests/google-servo-micro.json

You can copy the URLs from the LVFS UI -- and you only need one version in the device test, two would be nice, but one is still good.

Also, could you prefix your commit message with `telnk-dfu: ` please -- then it makes it easier for me to write NEWS files. Thanks!

Reply:
prefixed the commit title with 'telink-dfu:' and added plugins/telink-dfu/tests/telink-ryder-dongle.json
as well modified meson.build to include the .json file